packages/reports/src/Stores/Modules/Profit/profit-store.js

```diff
@@ -52,15 +51,6 @@ export default class ProfitTableStore extends BaseStore {
         });
     }
 
-    get total_profit() {
```

Comment:
This computed value is not used anywhere and it's returning an incorrect (non-rounded) JS value. E.g.: its calculating `(8.21 - 11.29) + (10.99 - 10)` as `-2.089999999999998` while the value we need is `-2.09`.